torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -30,20 +30,30 @@ export default async function handler(
 async function disableFlakyTests() {
     const [octokit, flaky_tests, issues] = await Promise.all([
         getOctokit(owner, repo), fetchFlakyTests(`${NUM_HOURS}`), fetchIssuesByLabel("skipped")]);
-    flaky_tests.forEach(async function (test) {
+
+    // If the test is flaky only on PRs, we should not disable it yet.
+    const flaky_tests_on_trunk = filterOutPRFlakyTests(flaky_tests);
```

Comment:
nit prefer camelcase

Reply:
Oops, I did this all throughout the file cuz I thought it was camel case for functions and snake for variables (and upon a glance I wasn't consistent anyway). Will fix in a separate PR